configs/example/gpufs/amd/AmdGPUOptions.py

```diff
@@ -247,16 +247,82 @@ def addAmdGPUOptions(parser):
         default="simple",
         help="register allocation policy (simple/dynamic)",
     )
+
     parser.add_argument(
         "--register-file-cache-size",
         type=int,
         default=0,
         help="number of registers in cache",
     )
+
     parser.add_argument(
         "--memtime-latency",
         type=int,
         # Set to a default of 41 from micro-benchmarks
         default=41,
         help="Latency for memtimes in scalar memory pipeline.",
     )
+
+    parser.add_argument(
+        "--dgpu",
+        action="store_true",
+        default=False,
+        help="Configure the system as a dGPU instead of an APU."
+        "The dGPU config has its own local memory pool and is not "
+        "coherent with the host through hardware.  Data is "
+        "transfered from host to device memory using runtime calls "
+        "that copy data over a PCIe-like IO bus.",
+    )
+
+    parser.add_argument(
+        "--gfx-version",
+        type=str,
+        default="gfx902",
+        choices=["gfx900", "gfx902", "gfx908", "gfx90a"],
```

Comment:
Missing `gfx942` (MI300).  Also unrelated to this patch, but the AMDGPUDevice uses a "device_name" argument which has the string value for these versions.  We should pick one or the other.  The `gfx*` choices are good in the sense that multiple devices can have the same gfx version (e.g., MI300X and MI300A are both gfx942).  On the other hand, more people would recognize MI300X vs. gfx942.

Reply:
See also: https://github.com/gem5/gem5/blob/develop/configs/example/gpufs/runfs.py#L127-L136 and https://github.com/gem5/gem5/blob/develop/configs/example/gpufs/mi300.py#L158

Reply:
Removed now